tests/templates/kuttl/spark-pi-public-s3/test.j2

```diff
@@ -0,0 +1 @@
+{{ test_scenario }}
```

Comment:
I think this is leftover from development and can be removed.

Reply:
Removed,